backend/routes/getMe_route.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+import { authToken } from '../utils/AuthenticateToken';
+import User from '../models/user';
+
+const router = express.Router();
+
+// fetch and return the currently authenticated user's information
+router.get('/me', authToken,async (req, res) => {
+    try{
+        await User.findById(req.user._id).select("-password");
+        res.status(200).send({success: true, message: "User Authenticated!"});
```

Comment:
I think you probably need to check if the user  from User.findById actually exists or not to see what response you should send, since mongoDB will return null if the user doesn't exist instead of throwing an error I believe